fe/utils.py

```diff
@@ -47,6 +47,8 @@ def convert_uIC50_to_kJ_per_mole(amount_in_uM):
     """
     return 0.593 * np.log(amount_in_uM * 1e-6) * 4.18
 
+def convert_uM_to_kJ_per_mole(amount_in_uM):
+    return 0.593 * np.log(amount_in_uM * 1e-6) * 4.18
```

Comment:
We could just convert all other examples to call the uM version instead of the uIC50 version since they're identical? (And rely on the end user to provide a binding potency in some unit of concentration?)

Reply:
(I ended up just refactoring this directly)